doc/content/devices/models/dragino-lht52/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Dragino LHT52"
+description: ""
+weight: 
+---
+
+{{< figure src="lht52.png" alt="Generic Node Sensor Edition" class="float plain">}}
+
+The Dragino [LHT52](https://www.dragino.com/products/temperature-humidity-sensor/item/199-lht52.html) is a temperature & humidity sensor and has a USB Type-C port to connect to external sensors such as an external temperature sensor.
+
+<!--more-->
+
+# Onboarding to {{% tts %}}
+
+The provisioning information, which are the **JoinEUI**, **DevEUI** and **AppKey** can be found on a sticker inside the box. 
+
+{{< figure src="lht52-prov-info.png" alt="LHT52 provisioning information" width="50%">}}
+
+The LHT52 does not support onboarding with a QR code. You must proceed directly to the options [**Select the end device in the LoRaWAN Device Repository**](#using-the-lorawan-device-repository) or [**Enter end device specifics manually**](#manually).
+
+## Using the LoRaWAN Device Repository
```

Comment:
I think this and content under Manually can be put under the same title, you can simplify it by just highlighting differences. It's not that this doesn't look good because it does, it just that we already have these titles [here](https://www.thethingsindustries.com/docs/devices/adding-devices/), so if we use the same ones here as well, we will have different content under same titles and by doing that, I think our search engine will eventually start throwing garbage.